omod/src/test/java/org/openmrs/module/attachments/rest/AttachmentResourceIntegrationTest.java

```diff
@@ -30,10 +35,8 @@ public String getUuidProperty() {
 	@Override
 	public void validateDefaultRepresentation() throws Exception {
 		super.validateDefaultRepresentation();
-		
 		assertPropEquals("dateTime", getObject().getDateTime());
 		assertPropEquals("comment", getObject().getComment());
-		assertPropEquals("complexData", getObject().getComplexData());
```

Comment:
Did you accidentally remove the above line?

Reply:
No... At the time I was still figuring out why that prop failed. That's now resolved in the upcoming commit. In short, until platform 2.1 complex obs `uuid`s were not part of the underlying saved files hence the failure to determine which file is attached to the complex obs for later versions.